src/forms/NumberInput.jsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import cx from 'classnames';
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+import Icon from '../media/Icon';
+
+export const DECREMENT_BTN_CLASS = 'decrementButton';
+export const INCREMENT_BTN_CLASS = 'incrementButton';
+
+/**
+ * @module NumberInput
+ */
+class NumberInput extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			value: props.value || '',
+		};
+
+		this.decrement = this.decrement.bind(this);
+		this.increment = this.increment.bind(this);
+		this.onBlur = this.onBlur.bind(this);
+		this.onChange = this.onChange.bind(this);
+		this.onFocus = this.onFocus.bind(this);
+	}
+
+	onBlur(e) {
+		if (document.activeElement !== this.fauxInputEl && document.activeElement !== this.decrementBtnEl && document.activeElement !== this.incrementBtnEl) {
```

Comment:
Yeah, Prettier formats it this way:

```js
onBlur(e) {
  if (
    document.activeElement !== this.fauxInputEl &&
    document.activeElement !== this.decrementBtnEl &&
    document.activeElement !== this.incrementBtnEl
  ) {
    this.fauxInputEl.classList.remove('focused');
  }
}
```

but I like the `formControls.every` recommendation - seems like a nice separation of data (`formControls`) and behavior (`if (...)`)